winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -104,22 +104,42 @@ namespace ROOT_NS.AI.MachineLearning
         //! Loads an ONNX model from a stream asynchronously.
         [remote_async]
         static Windows.Foundation.IAsyncOperation<LearningModel> LoadFromStreamAsync(Windows.Storage.Streams.IRandomAccessStreamReference modelStream);
+        //! Loads an ONNX model from a buffer asynchronously.
```

Comment:
curious what you think about grouping all of these into one (MachineLearningContract, 6) block instead of having four of them?

Reply:
Yeah technically I don't have problem with that, just thought it might make more sense to keep them together with their own kind

Reply:
Remember that this needs an associated change in the OS repo as well.